R/package.R

```diff
@@ -9,6 +9,11 @@
 #' @export
 as_pkgdown <- function(pkg = ".", override = list()) {
   if (is_pkgdown(pkg)) {
+    if (is.list(pkg$meta)) {
```

Comment:
Thanks for digging this up! Can you please use `modify_list()` instead since that already handles the `NULL` case, which should simplify this code a bit 😄

Reply:
I was just copying the version a couple of lines down; I just changed that one to `modify_list` too for consistency sake

Reply:
oh, also, I was checking if `x` rather than `y` is `null`; there are cases when `pkg$meta` is NULL. I just included a change to `modify_list` that checks both arguments so we can use that instead like you were suggesting.

Reply:
Perfect, thanks!